contracts/src/v0.8/ccip/capability/CCIPHome.sol

```diff
@@ -551,9 +552,10 @@ contract CCIPHome is OwnerIsCreator, ITypeAndVersion, ICapabilityConfiguration,
 
     ChainConfigArgs[] memory paginatedChainConfigs = new ChainConfigArgs[](endIndex - startIndex);
 
-    uint256[] memory chainSelectors = s_remoteChainSelectors.values();
+    // Use the at() function to get the chain selector at the given index without pagination which saves gas
+    // by avoiding the need to copy the entire value set
```

Comment:
nit: IMHO, the comment doesn't seem all that useful. I think the code speaks for itself!